src/modules/ethereum/account-abstraction/fetchAndAssignPaymaster.ts

```diff
@@ -11,40 +10,36 @@ import { GAS_PER_PUBDATA_BYTE_DEFAULT } from './constants';
 
 type NetworksSource = 'mainnet' | 'testnet';
 
-function adjustGas(value: ethers.BigNumberish): string {
-  return ethers.toQuantity(BigInt(value) + 50000n);
-}
-
 export function adjustedCheckEligibility(
-  params: Parameters<ZerionApiClient['paymasterCheckEligibility']>[0],
+  transaction: Parameters<ZerionApiClient['paymasterCheckEligibility']>[0],
   { source, apiClient }: { source: NetworksSource; apiClient: ZerionApiClient }
 ) {
-  const txCopy = { ...params };
-  const gas = getGas(txCopy);
-  invariant(gas, 'Tx param missing: {gas}');
-  const moreGas = adjustGas(gas);
-  txCopy.gas = moreGas;
-  return apiClient.paymasterCheckEligibility(txCopy, { source });
+  const transactionCopy = { ...transaction };
+  const gasLimitValue = getGas(transactionCopy);
+  invariant(gasLimitValue, 'Tx param missing: {gas}');
+  transactionCopy.gas = gasLimitValue;
+  transactionCopy.gasLimit = gasLimitValue;
```

Comment:
This is pointless if we no longer `adjustGas`? It's basically `tx.gas = tx.gas`

Reply:
Yes, let's just remove these lines instead